.travis.yml

```diff
@@ -5,10 +5,11 @@ python:
 before_install:
   - "export DISPLAY=:99.0"
   - "sh -e /etc/init.d/xvfb start"
+env:
+  - TOXENV=flake8
 install:
-  - "pip install flake8"
-  - "pip install -Ur requirements.txt"
-before_script: "flake8 ."
-script: "py.test -v --driver Firefox --variables variables.json tests"
```

Comment:
We still need to pass the variables file, though we can probably just do this in `tox.ini` as this file is in tree and contains no real secrets.